### DISCUSSION TITLE
SteamVR fail (error 476) showing everytime I load ALVR

### DISCUSSION THREAD

**Discussion by AlexTheGameAddict** [alvr-org/ALVR#1399] [Q&A]
I can't use ALVR on both my laptop/quest 2. Am I banned?
![wtf](https://user-images.githubusercontent.com/88726346/213870834-cc52d80c-9bf3-4e6c-a00f-a2abea6dac34.png)

**Comment by RubberFeng** [comment]
mee too.i change the configuration disorderly，and 
streaming successfully one time,then next time steamvr crash again.  hope someone have a answer.
by the way,my err msg is like "unable to fill the whole buffer"
